### ISSUE TITLE
Build fails on Julia 0.7.0, Win64 (user account)

### ISSUE THREAD

**Issue by PaulSoderlind** [JuliaWeb/LibCURL.jl#47]
When updating HDF5.jl today (15 Aug), the build of LibCURL failed (see below). It only seems to happen when logged in as a User, not as an Administrator.

``` 
(0.7) pkg> build HDF5
  Building LibCURL ─────→ `C:\Users\Me\.julia\packages\LibCURL\p0LUA\deps\build.log`
┌ Error: Error building `LibCURL`:
│ ERROR: LoadError: LoadError: Could not unpack C:\Users\Me\.julia\packages\LibCURL\p0LUA\deps\usr\downloads\Zlib.v1.2.11.x86_64-w64-mingw32.tar.gz into C:\Users\Me\.julia\packages\LibCURL\p0LUA\deps\usr
│ Stacktrace:
│  [1] error(::String) at .\error.jl:33
│  [2] #unpack#98(::Bool, ::Function, ::String, ::String) at C:\Users\Me\.julia\packages\BinaryProvider\pWVqZ\src\PlatformEngines.jl:600
│  [3] macro expansion at .\none:0 [inlined]
```

**Comment by krcools** [comment]
Could this perhaps be the symlink issue that also appeared at

https://github.com/bicycle1885/CodecZlib.jl/issues/24

In that case it can be solved by not using symlinks in the packaged binaries.

**Comment by omus** [comment]
cc: @staticfloat

**Comment by staticfloat** [comment]
That's a good idea @krcools; I bet if @omus changes the version of `ZlibBuilder` that he grabs to be `v1.0.2` instead of `v1.0.1`, this will work better.

**Comment by omus** [comment]
I'll make that change. Having the builders setup as packages would be nice avoid having to do these kinds of manual updates. Thanks for pointing that out.

**Comment by omus** [comment]
Can you check if this problem still occurs with these changes: https://github.com/JuliaWeb/LibCURL.jl/pull/48?

**Comment by zhaotianjing** [comment]
Same situation.
![image](https://user-images.githubusercontent.com/18593116/44185920-369f1280-a0cb-11e8-80dc-c19839fa8f05.png)
Win10 64bit user-login

**Comment by PaulSoderlind** [comment]
Same result here (using the current LibCURL#master, 16 Aug 08:16 GMT)

**Comment by krcools** [comment]
I think it is in fact exactly the same issue as for CodecZLib.  There, @bicycle1885 updated the builder. PR #49 refers to that new build.

**Comment by krcools** [comment]
Ah sorry you just did those changes. Not sure why people continue to see the bug. I had to start from a clean install. Maybe that's why...

**Comment by omus** [comment]
Thanks for verifying the new changes. I'll merge the PR and make a new tag. With the new tag I expect everyone will have a clean install.

**Comment by omus** [comment]
Should now be addressed in v0.3.1. The [release just needs to be registered](https://github.com/JuliaLang/METADATA.jl/pull/16924) so I'll close this issue. Feel free to open if the problem persists.
